text/metrics/0118-statsd-and-prometheus-exporters.md

```diff
@@ -0,0 +1,226 @@
+Types](https://prometheus.io/docs/concepts/metric_types).
+
+#### Prometheus instruments
+
+Prometheus Counter instruments are semantically identical to
+OpenTelemetry Counter instruments, including the Mnotonic property.
+They are exposed as a Cumulative Sum aggregation.
+
+Prometheus Gauge instruments have a number of uses, depending on
+whether they are used as an adding or as a grouping instrument.  They
+are exposed as a single data point equal to the the last value that
+was set.  Because Prometheus clients are stateful, Gauges support both
+`Set()` and `Add()` methods.  Generally, Prometheus Gauges used to
+`Add()` map into OpenTelemetry UpDownCounter instruments (maybe
+SumObserver, UpDownSumObserver), while Prometheus Gauges used to
+`Set()` map into OpenTelemetry ValueRecorder instruments (maybe
+ValueObserver).
+
+Prometheus Histogram instruments are exposed as Cumulative
+aggregations, defined by a number of bucketed counts, accumulated from
+the start of the process.  Prometheus Histogram instruments map into
+ValueRecorder instruments (maybe ValueObserver).  To configure a
+Histogram exposition in the Prometheus exporter, configure a Histogram
+Aggregator with the desired buckets.
+
+Prometheus Summary instruments are discouraged, as mentioned above,
+because they are not mergeable.  Uses of Prometheus Summary
+instruments map into OpenTelemetry ValueRecorder instruments (maybe
+ValueObserver).  To configure a Summary exposition in the Prometheus
+exporter, configure a Sketch Aggregator with the desired quantiles.
+
+#### OpenTelemetry Aggregator to Prometheus exposition
+
+The following table lists the mapping from Aggregator to Prometheus
+exposition format.  The "Typical Instruments" listed are the
+applicable OpenTelemetry instruments, for which the Prometheus mapping
+is sensible.
+
+| OpenTelemetry aggregator | Default Prometheus data type mapping | Export kind | Typical instruments | Notes |
+| -- | -- | -- | -- | -- |
+| Sum (Monotonic)       | Counter    | Cumulative | Counter(*), SumObserver(*) | |
+| Sum (Non-Monotonic)   | Gauge      | Cumulative | UpDownCounter(*), UpDownSumObserver(*) | |
+| LastValue             | Gauge      | Cumulative | ValueRecorder, ValueObserver, SumObserver, UpDownSumObserver | |
+| MinMaxLastSumCount    | Gauge      | Delta      | ValueRecorder(*), ValueObserver(*) | Expose the LastValue field as the Gauge |
+| Histogram             | Histogram  | Cumulative | ValueRecorder, ValueObserver | |
+| Sketch                | Summary    | Delta      | ValueRecorder, ValueObserver | |
+| Exact                 | Summary    | Delta      | ValueRecorder, ValueObserver | |
+
+Above, (*) denotes the default behavior of an OpenTelemetry
+instrument.
+
+### Statsd
+
+Statsd refers to a wire format for metrics. The mapping specified here
+refers to either the original Etsy protocol (a.k.a. plain Statsd) or
+the DataDog variation with labels added (a.k.a. DogStatsd).
+
+#### Statsd instruments
```

Comment:
@jmacd I believe this section is outlining the `statsdreceiver` metric type mappings? I see Summary in the metrics.proto, but I don't see Sketch. Can you share more details on the OpenTelemetry Sketch aggregator?

Reply:
There is no OTLP support for a sketch aggregator at this time. It is specified as an option for the SDK as a means of computing Summary values, although we would like to include OTLP support for sketches there is research and protocol description work to be done. Datadog has not published a serialization format for DDSketch as far as I know. I've proposed the use of a T-digest compatible format.

See this thread: https://github.com/open-telemetry/oteps/pull/113#discussion_r441885123